apps/web/pages/organizations/[id]/invoiceTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { FragmentType, graphql, useFragment } from '../../../frontend/generated/gql'
+
+export const InvoiceTableItemFragment = graphql(`
+  fragment InvoiceTableItem on Invoice {
+    id
+    invoiceDate
+    customerName
+    items {
```

Comment:
Hmm, I think `invoiceItems` is a different concept here. The idea, as I understand it, was to create something similar to `weekGrid`, where we have `invoices` as the main structure, and each `invoice` includes subfields (`invoiceItems`). These could potentially be expandable, just like the `weekGrid`. Renaming it to `invoiceItems` might cause confusion, as it would suggest that it only contains `invoiceItems` and not the broader `invoice` information. I hope that makes sense

Reply:
I think invoiceItems is the correct term.
Look at https://github.com/progwise/timebook/issues/955 => it is not subfields, it is more subrecords here.